esp/templates/program/manage_programs.html

```diff
@@ -18,7 +18,31 @@ <h2>Manage a Program</h2>
 <tbody>
   <tr>
     <td>
-        {% render_inline_qsd "manage/programs" %}
+        {% load render_qsd %}
+        {% inline_qsd_block "manage/programs" %}
+        ##Hello Fellow Admins!##
```

Comment:
I'm not sure how the new rich text editor will treat this markdown. Could you just use HTML here?

Reply:
boo. I guess.

Reply:
it seems to work fine? If you really want HTML I can rewrite it, but I think the markdown is more legible

Reply:
I just tested it, and things get really weird if you edit that QSD. Probably best to just have it all be HTML.